opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -172,9 +172,118 @@ message MetricDescriptor {
+    //      number of requests received over the interval of time t_0 to
+    //      t_0+1 with a value of 3.
+    //   6. A request is received, the system measures 1 request.
+    //   7. A request is received, the system measures 1 request.
+    //   8. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_0+1 to
+    //      t_0+2 with a value of 2.
+    DELTA = 2;
+
+    // CUMULATIVE is a metric whose values are the aggregation of
+    // successively made measurements from a fixed start time until the last
+    // reported measurement. This means that current values of a CUMULATIVE
+    // metric depend on all previous measurements since the start time.
+    // Because of this, the sender is required to retain this state in some
+    // form. If this state is lost or invalidated, the CUMULATIVE metric
+    // values MUST be reset and a new fixed start time following the last
+    // reported measurement time sent MUST be used.
+    //
+    // For example, consider a system measuring the number of requests that
+    // it receives every second and reports the sum of these requests as a
+    // CUMULATIVE metric:
+    //
+    //   1. The system starts receiving at time=t_0.
+    //   2. A request is received, the system measures 1 request.
+    //   3. A request is received, the system measures 1 request.
+    //   4. A request is received, the system measures 1 request.
+    //   5. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_0 to
+    //      t_0+1 with a value of 3.
+    //   6. A request is received, the system measures 1 request.
+    //   7. A request is received, the system measures 1 request.
+    //   8. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_0 to
+    //      t_0+2 with a value of 5.
+    //   9. The system experiences a fault and loses state.
+    //   10. The system recovers and resumes receiving at time=t_1.
+    //   11. A request is received, the system measures 1 request.
+    //   12. The 1 second collection cycle ends. A metric is exported for the
+    //      number of requests received over the interval of time t_1 to
+    //      t_0+1 with a value of 1.
+    CUMULATIVE = 3;
+  }
+
+  // temporality is the Temporality of values this metric has.
+  Temporality temporality = 5;
+
+  // Monotonic is a refinement of the values a metric has. It defines the
+  // relationship values of successively reported metrics have
+  // (non-increasing, non-decreasing, or unknown). This is a refinement of
+  // the metric values that can be useful for a receiver in understanding
+  // how to deal with discontinuities in the data (i.e.  calculating
+  // derivates of the data without introducing artifacts from a reset).
```

Comment:
Also it is used to determine if calculating a rate makes sense or not (for example if a metric is not monotonic increasing then the rate, if reported t0->v0 and t1->v1, (v1 - v0) / (t1 - t0) may not be accurate because the value may jumped and come back to v1.

Reply:
What you just describe is a derivative (though I'm guessing you already knew that). I like the more general term derivative, but are you suggesting a preference for rate?